litmus-portal/frontend/src/views/Analytics/Overview/index.tsx

```diff
@@ -252,14 +252,14 @@ const Overview: React.FC = () => {
                 });
               }}
             >
-              <Typography>Add dashboard</Typography>
+              <Typography>Create dashboard</Typography>
```

Comment:
```suggestion
              <Typography>Create a dashboard</Typography>
```

Reply:
we have removed a from the text @vanshBhatia-A4k9 during design updated and UX fixes